packages/wrangler/e2e/dev-registry.test.ts

```diff
@@ -27,6 +29,114 @@ async function fetchJson<T>(url: string, info?: RequestInit): Promise<T> {
 	);
 }
 
+describe("unstable_dev()", () => {
+	let parent: string;
+	let child: string;
+	let workerName: string;
+
+	beforeEach(async () => {
+		workerName = generateResourceName("worker");
+
+		parent = makeRoot();
+
+		await seed(parent, {
+			"wrangler.toml": dedent`
+					name = "app"
+					compatibility_date = "2023-01-01"
+					compatibility_flags = ["nodejs_compat"]
+
+					[[services]]
+					binding = "WORKER"
+					service = '${workerName}'
+			`,
+			"src/index.ts": dedent/* javascript */ `
+					export default {
+						async fetch(req, env) {
+							return new Response("Hello from Parent!" + await env.WORKER.fetch(req).then(r => r.text()))
+						},
+					};
+					`,
+			"package.json": dedent`
+					{
+						"name": "app",
+						"version": "0.0.0",
+						"private": true
+					}
+					`,
+		});
+
+		child = await makeRoot();
+		await seed(child, {
+			"wrangler.toml": dedent`
+						name = "${workerName}"
+						main = "src/index.ts"
+						compatibility_date = "2023-01-01"
+				`,
+			"src/index.ts": dedent/* javascript */ `
+					export default {
+						fetch(req, env) {
+							return new Response("Hello from Child!")
+						},
+					};
+					`,
+			"package.json": dedent`
+						{
+							"name": "${workerName}",
+							"version": "0.0.0",
+							"private": true
+						}
+						`,
+		});
+	});
+
+	async function runInNode() {
+		await seed(parent, {
+			"index.mjs": dedent/*javascript*/ `
+					import { unstable_dev } from "${WRANGLER_IMPORT}"
+					import { setTimeout } from "node:timers/promises";
+
+					const childWorker = await unstable_dev(
+						"${child.replaceAll("\\", "/")}/src/index.ts",
+						{
+							configPath: "${child.replaceAll("\\", "/")}/wrangler.toml",
+							experimental: {
+								disableExperimentalWarning: true,
+							},
+						}
+					);
+
+					const parentWorker = await unstable_dev(
+						"src/index.ts",
+						{
+							configPath: "wrangler.toml",
+							experimental: {
+								disableExperimentalWarning: true,
+							},
+						}
+					);
+
+					await setTimeout(2000)
+
+					console.log(await parentWorker.fetch("/").then(r => r.text()))
```

Comment:
is this `log` intentional here?

Reply:
In this case, yes. This is a `.js` file that is run using `node` on line 125. The return value of the `runInNode()` function is the `stdout` of the node subprocess, which is used to assert against further down the file in the test cases.

Reply:
yeah, I suspected this might be the reason why it's there, but making sure